iron-ajax.html

```diff
@@ -238,6 +239,16 @@
         }
       },
 
+      /**
+       * Length of time in milliseconds to debounce multiple requests.
+       */
+      debounceFor: {
```

Comment:
Nice, this is a good idea. I'm would like to propose that the property be renamed `debounceDuration`. This clarifies the type of the value, and also matches language we use elsewhere in the element catalogue.